### PR TITLE
Enable GeoJSON responses from the facilities nearby endpoint

### PR DESCRIPTION
**Pull request by rtravitz**
## Description of change

This PR addresses [#2291](https://github.com/department-of-veterans-affairs/vets-contrib/issues/2291). It enables GeoJSON responses for the new Facilities API `/nearby` endpoint. Changes to the serialization are handled in a new `geo_serializer_v1.rb` file. This is very similar to the pre-existing v0 serializer (only one line changes). At some point, we may want to refactor how we handle versioning our serializers in a DRY-er way. 

## Testing done

Tests were enabled for GeoJSON responses in `nearby_request_spec.rb`, including a new test to enforce schema changes around health service wait times between v0 and v1. 

#### Unique to this PR

- [x] Create a new serializer for GeoJSON
- [x] Turn tests on in modules/va_facilities/spec/requests/nearby_request_spec.rb

#### Applies to all PRs

- [x] Appropriate logging
- [x] Swagger docs have been updated, if applicable
- [x] Provide link to originating GitHub issue, or connected to it via ZenHub
- [x] Does not contain any sensitive information (i.e. PII/credentials/internal URLs/etc., in logging, hardcoded, or in specs)

### CODE REVIEW

File: modules/va_facilities/spec/requests/nearby_request_spec.rb [hunk 1]
```diff
@@ -146,55 +146,88 @@ def query_params(url)
...
+        expect(json['features'].length).to eq(0)
+        expect(response.headers['Link']).to be_present
+        parsed = parse_link_header(response.headers['Link'])
+        expect(parsed['self']['page']).to eq(['1'])
+        expect(parsed['first']['page']).to eq(['1'])
+        expect(parsed['last']['page']).to eq(['1'])
+      end
+    end
+
+    it 'responds with wait times as part of health services in v1' do
```
Comment (edmkitty) [inline modules/va_facilities/spec/requests/nearby_request_spec.rb]: How painful would it be to add a test case for a facility with no health services? Is that a thing?

Comment (rtravitz) [reply modules/va_facilities/spec/requests/nearby_request_spec.rb]: It turns out there are 46 health facilities that list no health services—a few list no services at all. I wonder if something like that should go in a more general test that doesn't require the cassette like this `/nearby` endpoint.

**Comment by davem-va** [comment]
@rtravitz In the near term, will be moving our source of health services to one from CDW and then using access to care exclusively to determine services provided by a facility.  Currently we use an arcGIS feed to determine primary care, dentistry, and mental health services and access to care for the rest.  This may or may not remedy the 46.

Even if it does not, there is a larger effort in VA to determine an ADS for all services provided by a VHA health facility.  Even access to care only focuses on the "high encounter" ones.  This will ultimately fix any issues that remain after the near term work.

**Comment by rtravitz** [comment]
@davem-va This extra context is helpful! 
@edmkitty I'm looking into an extra case to make sure nothing goes wrong with the way the data currently exists. But also check out Dave's comment above!

File: modules/va_facilities/spec/requests/nearby_request_spec.rb (hunk 1 above)
Comment (rtravitz) [reply modules/va_facilities/spec/requests/nearby_request_spec.rb]: @edmkitty this was painful, but I did finally get a spec added.

**Pull request merged**